components/card/card.css

```diff
@@ -2,101 +2,160 @@
 :root {
   --card-shadow-color: #e6e3e3;
   --card-text-color: #9b9b9b;
-  --card-link-color: #e073d9;
-  --card-link-hover-color: #fff1fe;
+  --card-discount-color: green;
```

Comment:
can you use hex code here for green?

Reply:
any specific reason ?

Reply:
See you have been using hex code for every other variable, so it is good practice to use hex code for this one too...

Reply:
ohh ok...will do